libs/data-mapper/src/lib/components/testMapPanel/TestMapPanel.tsx

```diff
@@ -233,7 +233,7 @@ export const TestMapPanel = ({ mapDefinition, isOpen, onClose }: TestMapPanelPro
       </Stack>
     );
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [fileXslt, isMismatchedXslt]);
+  }, [fileXslt, testMapInput, isMismatchedXslt]);
```

Comment:
oh, the reason we did not see this earlier is because of the above line that suppresses the error. Can you remove it?

Reply:
removed it

Reply:
<img width="776" alt="image" src="https://github.com/Azure/LogicAppsUX/assets/144840522/086694ca-4537-4a14-a6f8-4c06cddba25d">

Disabling the suppress error requires that I include everything in the dependency list including testMap. It seems like including testMap in the dependency list gives this error, which is why the suppress error script was there. I brought back the suppress error dialog and made sure to add all the other dependencies suggested. Would this work?

Reply:
@refortie would you have any insight?

Reply:
Might need to make testMap into a useCallback, which should prevent it from re-rendering all the time

Reply:
@refortie Got it, changed it accordingly. I tested that things work as expected.